### ISSUE TITLE
oauth.tools for demo app and redirect url

### ISSUE THREAD

**Issue by jschoch** [tanguilp/asteroid#6]
It looks like you need to add the oauth.tool redirect url to lib/asteroid/application.ex to get it working.

**Comment by tanguilp** [comment]
Which branch are you talking about?

The redirect URIs a present on the `demo_auth_workflow_gigalixir` branch (https://github.com/tanguilp/asteroid/blob/demo_auth_workflow_gigalixir/lib/asteroid/application.ex#L75).

On the `demo_auth_workflow_gigalixir` it is not, because 1) the discovery document is not reachable from oauth2.tools (which is NOT a full-javascript app, the backend does the heavy work) and 2) as far as I remember oauth2.tools doesn't want to redirect to `localhost` (but not 100% sure about it, would worth checking again)

**Comment by jschoch** [comment]
I was referring to the demo_auth_workflow branch.  discovery does not work but you can input the values and send requests after you add the callback.

**Comment by tanguilp** [comment]
Indeed, but the documentation doesn't refer to oauth.tools in the "Run locally" section. And at that point, I guess the developer would have enough knowledge to add the callbacks himself?

Still considering adding it or not.
